### ISSUE TITLE
jfx-luna value error

### ISSUE THREAD

**Issue by skeleturtle** [juno-fx/report#194]
We are getting hit once again with that error with the 'value_' 

![image](https://github.com/juno-fx/report/assets/76455095/6dcf17fd-2a95-4f28-8e92-56f9ed7cf8a3)

**Comment by aldmbmtl** [comment]
Looking into it

**Comment by skeleturtle** [comment]
Switching assignment to @oneBlueFish since he is working on doing away with the auto generated code.

**Comment by oneBlueFish** [comment]
I have updated `jfx-luna` off of the auto generated services, this should help with this issue

There is just a little clean up i would like to do

@skeleturtle , if you would like to try the code out, pull down the `removing_auto_code` branch on `jfx-luna` 

All the pytests passed, so you shouldn't run into any issues, but if you do, let me know :)

**Comment by skeleturtle** [comment]
> I have updated `jfx-luna` off of the auto generated services, this should help with this issue
> 
> There is just a little clean up i would like to do
> 
> @skeleturtle , if you would like to try the code out, pull down the `removing_auto_code` branch on `jfx-luna`
> 
> All the pytests passed, so you shouldn't run into any issues, but if you do, let me know :)

Getting errors on just about every call I make. to jfx-luna. 

Example 1:
- using the meta_helper to get_all_metas we are now returning dict objects for each meta, where previously we were getting the outMeta objects.
- using version_helper to get_version_by_id throws an error that value is not a valid dict

**Comment by aldmbmtl** [comment]
@oneBlueFish do you need my help on this one?

**Comment by skeleturtle** [comment]
@oneBlueFish 

Adding some of what we talked about on discord here for visibility.

I pushed another update to my branch. We ran into the junoFiles file_type issue again. What I noticed was the JunoF jfx-luna class was the only one that wasn't using the the rawOutData from our models. It was inheriting our in junoFile model. This is what was causing the descrepencies when casting our outJunoFile into a JunoFile class, and us trying to work around.
